config/BGQ_Clang++11_ToolChain.cmake

```diff
@@ -1,6 +1,8 @@
 set(CMAKE_C_COMPILER mpiclang) 
 set(CMAKE_CXX_COMPILER mpiclang++11)
 
+set(COMPILING_ON_BGQ TRUE)
```

Comment:
You probably has checked. This doesn't need to be cached, right? I mean only need 1 cmake.

Reply:
The toolchain file gets called many times, including before the regular CMakeLists.txt is processed.  I don't think this needs to be cached.